packages/shared/styles/components/SfBanner.scss

```diff
@@ -34,11 +34,20 @@ $banner--slim__title-font-size: $h1-font-size-desktop !default;
   background-repeat: no-repeat;
   background-position: bottom left;
   background-size: cover;
+  background: --background-color;
+  @if --m {
```

Comment:
You can't use SCSS function for CSS variables :) because you transpile SCSS to CSS before set CSS variable :) and remember to use var(--variable-name)